src/app/services/wallet.service.ts

```diff
@@ -176,53 +173,11 @@ export class WalletService {
     return this.apiService.get('pendingTxs');
   }
 
-  recent(): Observable<any[]> {
-    return this.recentTransactions.asObservable();
-  }
-
-  refreshBalances() {
```

Comment:
I think that `refreshBalances` should not have been deleted. This is due to two things:

- The desktop wallet uses `refreshBalances`. Using refreshBalances would help to maintain the consistency between the two wallets. At this time the web wallet uses `loadBalances` instead.

- `loadBalances` seems to have several important problems, such as that it can send too many parameters through the URL and throw errors because of that.

A more in-depth review should be done to decide which of the two functions will be used at the end (probably with modifications). Also `retrieveAddressBalance` and `retrieveWalletBalance` would not be needed if `refreshBalances` is deleted.